index.html

```diff
@@ -1,20 +1,970 @@
+            </ul>
+            <div class="col-5  hamburger-menu">
+                <a href=""><img src="img/menu-hamburger.svg" alt="hamburger menu"></a>
+            </div>
+        </div>
+
+
+
+    </header>
+
+
+    <main>
+        <section class="side-bars container">
+            <div class="row">
+                <a href="#first">
+                    <p class="side-bar-dew">DEW FESTIVAL MAY 28</p>
+                </a>
+
+                <ul class="col-md-11 social-bar">
+
+                    <li class="d-md-block"><a href="https://twitter.com/wix"><img
+                                src="https://static.wixstatic.com/media/c7d035ba85f6486680c2facedecdcf4d.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/c7d035ba85f6486680c2facedecdcf4d.webp"
+                                alt="Twitter icon"></a></li>
+                    <li class="d-md-block"><a href="https://www.youtube.com/user/Wix"><img
+                                src="https://static.wixstatic.com/media/78aa2057f0cb42fbbaffcbc36280a64a.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/78aa2057f0cb42fbbaffcbc36280a64a.webp"
+                                alt="YouTube icon"></a></li>
+                    <li class="d-md-block"><a href="https://www.facebook.com/wix"><img
+                                src="https://static.wixstatic.com/media/0fdef751204647a3bbd7eaa2827ed4f9.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/0fdef751204647a3bbd7eaa2827ed4f9.webp"
+                                alt="Facebook icon"></a></li>
+                    <li class="d-md-block"><a href="https://www.instagram.com/wix/"><img
+                                src="https://static.wixstatic.com/media/01c3aff52f2a4dffa526d7a9843d46ea.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/01c3aff52f2a4dffa526d7a9843d46ea.webp"
+                                alt="Instagram icon"></a></li>
+                    <li class="d-md-block"><a href="https://soundcloud.com/"><img
+                                src="https://static.wixstatic.com/media/03ea2af965014a3eb32df628a4c18c0e.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/03ea2af965014a3eb32df628a4c18c0e.webp"
+                                alt="SoundCloud icon"></a></li>
+                </ul>
+
+                <a href="#line-up">
+                    <p class="side-bar-line">LINE UP</p>
+                </a>
+
+            </div>
+            <div class="row">
+                <a href="#info">
+                    <p class="side-bar-info">INFO</p>
+                </a>
+                <a href="#buy-ticket">
+                    <p class="side-bar-buy">BUY TICKETS</p>
+                </a>
+
+
+            </div>
+
+
+        </section>
+        <section class="container-fluid first-section" id="first">
+            <div class="row justify-content-center">
+                <div class="relative">
+
+                    <svg class="col-12 col-md-8 offset-md-2 big-img-ab" data-bbox="18.416 22.603 162.965 154.895"
```

Comment:
If you inspect this image in mobile mode, you'll see that because of the rotation, it's become wider than the page and is pushing the sides out (when incpecting you can see the element highlighted as a blue box on the page - observe how why that blue box is!). This is why there is a horizontal scroll on your mobile page.
To fix this, set `overflow: hidden;` on the section. It will essentially chop off anything that is overlapping outside of the desired size.